### ISSUE TITLE
Investigate intent attributes for accessibility 

### ISSUE THREAD

**Issue by Wyatt915** [Wyatt915/treeblood#26]
See https://w3c.github.io/mathml-docs/intent-core-properties/

Many of these should be able to be automatically generated without too much pain. The goal of TreeBlood is first and foremost increasing the accessibility of mathematics, so this should be a major priority. However, the spec for the intent attribute is still being developed so the focus should be on a flexible system to emit intents.
